test/types/Model.ts

```diff
@@ -9,3 +10,30 @@ const shouldFailWithNumberAsName = dynamoose.model(1);
 const shouldFailWithBooleanAsName = dynamoose.model(true);
 
 const shouldSucceedWithOnlyPassingInName = dynamoose.model("User");
+
+interface User extends Document {
+	id: string;
+	name: string;
+}
+
+const UserModel = dynamoose.model<User>(
+	"User",
+	new dynamoose.Schema({
+		"id": String,
+		"name": {
+			"type": String,
+			"index": {
+				"global": true
+			}
+		},
+		"age": Number
+	})
+);
+
+UserModel.create({"id": "1", "name": "john", "age": 25});
+
+UserModel.get({"id": "1"});
+
+UserModel.query("name").eq("john").exec();
+
+UserModel.scan().exec();
```

Comment:
New line at end of file. See #950.

```suggestion
UserModel.scan().exec();

```